### PR TITLE
Allow manual deployment

### PR DESCRIPTION
**Pull request by jeylau**
It is frustrating to see the publication of new releases to PyPi fail because some GitHub's CI tests do not pass (at random..., while they all pass locally). With the current changes, one could manually trigger the `deploy` step (either from the GitHub UI or CLI), which I find convenient.

### CODE REVIEW

**Review by n-poulsen — APPROVED** [review]
Looks good to me! No way to test this before we merge right?

**Comment by jeylau** [comment]
Screenshot 2024-02-20 at 4 02 16 PM
Let me look; it should fail to publish anyway since the tag v0.2.1.4 had already been used.

**Comment by jeylau** [comment]
@n-poulsen, can I merge this one and then publish a new version?

**Pull request merged**
